src/lib/y2network/interface_config_builder.rb

```diff
@@ -61,14 +61,22 @@ def newly_added?
       Yast::LanItems.operation == :add
     end
 
+    # change internal config keys.
```

Comment:
np: we should be consistent in the style: "change" or "changes"? In the next method (`[]`) you are using "gets" and, later, (`save`) "save". :)

Reply:
ideally it should be changes, similar as in rspec.